phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -3593,10 +3593,11 @@ public MutationState addColumn(PTable table, List<ColumnDef> origColumnDefs,
 
                 int position = table.getColumns().size();
 
-                List<PColumn> currentPKs = table.getPKColumns();
-                if (numCols > 0) {
+                boolean addPKColumns = columnDefs.stream().anyMatch(ColumnDef::isPK);
```

Comment:
nice one-liner change!

Reply:
Do you mean I should remove the `addPKColumns` variable and change the if statement to something like the following:
```
                if (columnDefs.stream().anyMatch(ColumnDef::isPK)) {
```

Reply:
I wasn't suggesting anything. I rather appreciated your change :)

Reply:
Sure. Thank you for reviewing. I will change the patch for your review.